easy-to-use-api/klips-url-generator/src/App.tsx

```diff
@@ -108,51 +113,61 @@ const App: React.FC = () => {
           region={region}
           wktGeom={wktGeom}
           onDrawEnd={onDrawEnd}
-          // onDrawStart={onDrawStart}
         />
       case 'video':
         return <VideoComponent
           geoJsonGeom={geoJsonGeom}
           region={region}
           onDrawEnd={onDrawEnd}
-          // onDrawStart={onDrawStart}
         />
       case 'warning':
         return <WarningComponent
           wktGeom={wktGeom}
           geoJsonGeom={geoJsonGeom}
           region={region}
           onDrawEnd={onDrawEnd}
-          // onDrawStart={onDrawStart}
         />
     }
   }
 
   return (
     <div className='App'>
-      <MapContext.Provider value={map}>
-        <div className='output-wrapper'>
-          <div className='header'>
-            <img src="https://www.klips-projekt.de/wp-content/uploads/2021/02/SAG_KLIPS-Logo_Jan21.png" alt="KLIPS Logo"></img>
-            <h2>Widget URL Generator</h2>
+      <ConfigProvider
+        theme={{
+          algorithm: theme.defaultAlgorithm,
+          token: {
+            colorPrimary: '#e45f24',
```

Comment:
Could that me moved to the constants file?

Reply:
Thank you, I moved the token to the constants file.